src/test/common/ios/xcodeproj.test.ts

```diff
@@ -11,22 +11,22 @@ suite("xcodeproj", function() {
     suite("commonContext", function() {
         test("should look in the correct location for xcodeproj files and return one", function() {
             const projectRoot = path.join("/", "tmp", "myProject");
-            const extension = "xcodeproj";
             const testFiles = ["foo.xcodeproj"];

```

Comment:
You may want to utilize the new type for expect and actual comparison here. But it's a minor issue. NBD.